hw/mcu/stm/stm32_common/src/hal_timer.c

```diff
@@ -28,6 +28,11 @@
 #include "mcu/stm32_hal.h"
 #include "stm32_common/stm32_hal.h"
 
+#if MYNEWT_VAL(MCU_STM32L0)
+void TIM_Base_SetConfig(TIM_TypeDef *TIMx, TIM_Base_InitTypeDef *Structure);
+void TIM_CCxChannelCmd(TIM_TypeDef* TIMx, uint32_t Channel, uint32_t ChannelState);
```

Comment:
`*` used inconsequently in those two lines

Reply:
I fixed the inconsistency on latest rebase (this was just copy/paste from stm32cube because those functions were made `static`). Thanks!